src/main/java/codeu/model/data/User.java

```diff
@@ -21,20 +21,29 @@
 public class User {
   private final UUID id;
   private final String name;
+  private final String password;
   private final Instant creation;
 
   /**
    * Constructs a new User.
    *
    * @param id the ID of this User
    * @param name the username of this User
+   * @param password the password of this User
    * @param creation the creation time of this User
    */
-  public User(UUID id, String name, Instant creation) {
+  public User(UUID id, String name, String password, Instant creation) {
     this.id = id;
     this.name = name;
+    this.password = password;
     this.creation = creation;
   }
+  /**
```

Comment:
add a newline right above this comment to separate this from the constructor above

Reply:
Done